examples/stable_diffusion/config_vae_decoder.json

```diff
@@ -7,7 +7,7 @@
         "io_config": {
             "input_names": [ "latent_sample", "return_dict" ],
             "output_names": [ "sample" ],
-            "dynamic_axes": { "latent_sample": { "0": "batch", "1": "channels", "2": "height", "3": "width" } }
+            "dynamic_axes": { "latent_sample": { "0": "decoder_batch", "1": "decoder_channels", "2": "decoder_height", "3": "decoder_width" } }
```

Comment:
The linter is complaining about the formatting for these changes. 

Instructions for running the liner and applying the changes can be found at https://github.com/microsoft/Olive/blob/main/CONTRIBUTING.md#linting